src/bo4e_generator/__main__.py

```diff
@@ -37,13 +37,21 @@ def generate_bo4e_schemas(input_directory: Path, output_directory: Path):
     help="Output directory for the generated python files.",
     required=True,
 )
+@click.option(
+    "--pydantic-v1/--pydantic-v2",
+    "-p",
```

Comment:
und wie würde man in der abgekürzten Form angeben?
`-p 1` vs `-p 2` oder `-p v1` ?

Reply:
https://github.com/Hochfrequenz/BO4E-Python-Generator/pull/12/commits/45c8dfa054ec6978e0a723acbb1fc6e904615ce0